docs/babel.mdx

```diff
@@ -2,26 +2,26 @@
 title: 'Babel Plugin'
 ---
 
-`@emotion/babel-plugin` is highly recommended. All of the options that can be provided to `@emotion/babel-plugin` are documented in [`@emotion/babel-plugin`'s README](https://github.com/emotion-js/emotion/tree/main/packages/babel-plugin).
+`@emotion/babel-plugin` is highly recommended. All of the options that can be provided to `@emotion/babel-plugin` are documented in [`@emotion/babel-plugin`'s README](/docs/@emotion/babel-plugin).
```

Comment:
this URL (and probably some other similar ones) is not working within the repository itself now, so when viewing `.mdx` alone you can't navigate freely. In the old version we had some smart rewrites to make it work for the site itself (since we can't affect how it's resolved within GitHub itself)

Reply:
Here's the code you are referring to: https://github.com/emotion-js/emotion/blob/95726a905133362b6e58a4c0f9c1e97385a393f8/site/plugins/gatsby-remark-fix-links/index.js

It did not rewrite GitHub links like this one, so this link would take you from the documentation site to the `.mdx` file in GitHub. You can try using [an old deploy preview](https://deploy-preview-2819--emotion.netlify.app/docs/babel).

So I guess the ideal solution would be to add new URL rewriting code so this links to the documentation site or GitHub depending on the context.

Reply:
Ah, good catch - it never worked correctly for this case here. But ye - would be best if GitHub links would just work, either by using absolute links with the `/main/` segment in them or using relative paths and we would rewrite those when building the site.